code/modules/reagents/chemistry_properties/prop_neutral.dm

```diff
@@ -519,6 +519,31 @@
 /datum/chem_property/neutral/hyperthrottling/process_critical(mob/living/M, potency = 1, delta_time)
 	M.apply_effect(potency * delta_time, PARALYZE)
 
+/datum/chem_property/neutral/encephalophrasive
+	name = PROPERTY_ENCEPHALOPHRASIVE
+	code = "ESP"
+	description = "Drastically increases the amplitude of Gamma and Beta brain waves, allowing the host to broadcast their mind."
+	rarity = PROPERTY_LEGENDARY
+	category = PROPERTY_TYPE_STIMULANT
+	value = 8
+
+/datum/chem_property/neutral/encephalophrasive/on_delete(mob/living/chem_host)
+	..()
+
+	chem_host.pain.recalculate_pain()
+	remove_verb(chem_host, /mob/living/carbon/human/proc/psychic_whisper)
+	to_chat(chem_host, SPAN_NOTICE("The pain in your head subsides, and you are left feeling strangely alone."))
+
+/datum/chem_property/neutral/encephalophrasive/process(mob/living/chem_host, potency = 1, delta_time)
+	chem_host.pain.apply_pain(1 * potency)
+	add_verb(chem_host, /mob/living/carbon/human/proc/psychic_whisper)
```

Comment:
This shouldn't be added every tick; ideally add it once when the chem first enters someone

Reply:
Fair. I'll try to figure out how to do that. It's handled the same way in other chems so I cant just look at the pattern

Reply:
Changed thanks to the help of diegoflores31
Pain is still processed every tick
adding the verb is handled upon initial ingestion